src/encoded/tests/test_types_series.py

```diff
@@ -8,7 +8,8 @@ def base_reference_epigenome(testapp, lab, award):
         'lab': lab['uuid'],
         'status': 'submitted'
     }
-    return testapp.post_json('/reference-epigenomes', item, status=201).json['@graph'][0]
+    print (testapp.post_json('/reference_epigenome', item))
```

Comment:
This print looks like it should be removed.

Reply:
removed